### PR TITLE
Update styling of examples

### LINKED ISSUE #27: Update Examples with IBM Design Language tabs

### PR DESCRIPTION
**Pull request by matthewgallo**
Closes #27 

#### Changelog

**New**

- Updated scatterplot example tab styling to be more consistent and aligned with the IBM Design Language (and accessible).

### CODE REVIEW

File: examples/js/scatterplot.js [hunk 1]
```diff
@@ -754,4 +754,38 @@ function Scatterplot(el, component, svgClass) {
     return str.replace(' ', '').toLowerCase()
   }
 }
+
+  var scatterplot1Tab1 = document.querySelector('.scatterplot-1-tab-1');
```
Comment (seejamescode) [inline examples/js/scatterplot.js]: I would really love to see [this css-only way](https://codepen.io/rm/pen/ldhon) implemented before merging!

**Review by seejamescode — CHANGES_REQUESTED** [review]

File: examples/js/scatterplot.js (hunk 1 above)
Comment (matthewgallo) [reply examples/js/scatterplot.js]: Ok, will work on changing this!

Comment (joshblack) [reply examples/js/scatterplot.js]: Is that example accessible @seejamescode? I thought you require JS for UI patterns like that one, in order to make things accessible? 🤔

File: examples/css/style.css [hunk 2]
```diff
@@ -57,26 +60,28 @@ body {
...
+.scatterplot .erase:hover .erase-fill-element,
+.scatterplot .erase:focus .erase-fill-element {
+  fill: #5a3ec8;
 }
 
 .scatterplot .erase svg {
   cursor: pointer;
 }
 
 .scatterplot .menuEl, .scatterplot .menuEl2 {
```
Comment (joshblack) [inline examples/css/style.css:76]: Is there a convention for things that are `kebab-cased` and `camelCased`? noticed some of the selectors follow the former, some the latter.

Comment (matthewgallo) [reply examples/css/style.css:76]: In the original code from Accurat, the menu elements contained the class `menuEl` and `menuEl2`. We can change it to stay consistent with `kebab-cased` if that makes sense.

Comment (joshblack) [reply examples/css/style.css:76]: Up to you! Just was curious.

File: examples/js/scatterplot.js (hunk 1 above)
Comment (seejamescode) [reply examples/js/scatterplot.js]: Only thing I see lacking for it to be accessible is when:
`.two:hover ~ hr {`

should also include `focus` along with `hover`.

Comment (joshblack) [reply examples/js/scatterplot.js]: I meant more like, `role="tablist"`, `role="tab"`, `aria-labbeledby`, `aria-controlled`, `aria-selected`, etc

Comment (matthewgallo) [reply examples/js/scatterplot.js]: I've updated it to the css only way, however, upon hovering the third tab the line starts from the first tab and then moves to the third tab. Whereas, in the js implementation the line starts from whatever tab you first hover on.
scatterplot_tabs_animation

Comment (seejamescode) [inline examples/js/scatterplot.js]: Sorry for sending you a bad example. However, now you know how to do the hover transition with only css. So try keep using css for those transitions, but just trigger the transitions from the tablist classname during tab onClicks.

Also, Josh is right about those labels.

Comment (seejamescode) [inline examples/js/scatterplot.js]: Sorry for sending you a bad example. However, now you know how to do the hover transition with only css. So try keep using css for those transitions, but just trigger the transitions from the tablist classname during tab onClicks.

Also, Josh is right about those labels.

Comment (seejamescode) [inline examples/js/scatterplot.js]: Sorry for sending you a bad example. However, now you know how to do the hover transition with only css. So try keep using css for those transitions, but just trigger the transitions from the tablist classname during tab onClicks.

Also, Josh is right about those labels.

Comment (seejamescode) [reply examples/js/scatterplot.js]: Sorry for sending you a bad example. However, now you know how to do the hover transition with only css. So try keep using css for those transitions, but just trigger the transitions from the tablist classname during tab onClicks.

Also, Josh is right about those labels.

**Comment by joshblack** [comment]
What's the status of this PR?

**Comment by matthewgallo** [comment]
I'm going to work on it this afternoon and fix everything specified above.

**Comment by seejamescode** [comment]
Hey @matthewgallo, did you get those changes done for the tabs?

**Comment by matthewgallo** [comment]
Yes, I did...however changing the tabs to radio buttons (like the example we had looked at before) stopped the d3 code from changing the active class on each tab. Besides that, this new solution seems best, all css.

**Pull request closed without merging**